src/cdk/drag-drop/directives/drag.ts

```diff
@@ -330,9 +330,11 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
 
   /** Syncs the root element with the `DragRef`. */
   private _updateRootElement() {
-    const element = this.element.nativeElement;
+    const element = this.element.nativeElement as HTMLElement;
+    // Comment tag doesn't have closest method, so use parent's one.
+    const closestFn = (element.closest || element.parentElement?.closest)?.bind(this);
```

Comment:
Sorry for nitpicking, but I think that you'll have to push again anyway because the CI didn't run.

Using `bind` here is a little wasteful since it creates a new function. It would be better to use `apply` or `call`.

Reply:
@crisbeto thanks! 🙂